index.html

```diff
@@ -4292,6 +4306,493 @@ <h3>Proof Formats</h3>
       </section>
     </section>
 
+    <section class="normative">
+      <h2>Algorithms</h2>
+
+      <p>
+This section contains algorithms that can be used by implementations to
+perform common operations.
+      </p>
+
+      <p class="note" title="Implementers can include additional checks, warnings, and errors.">
+Implementers are advised that the algorithms in this section contain the bare
+minimum set of checks used by implementations to test conformance to this
+specification. Implementations are expected to provide additional checks that
+report helpful warnings for developers to help debug potential issues.
+Similarly, implementations are likely to provide additional checks that
+could result in new types of errors being reported in order to stop harmful
+content. Any of these additional checks might be integrated into future
+versions of this specification.
+      </p>
+
+      <section class="normative">
+        <h3>Verification</h3>
+
+        <p>
+This section contains an algorithm that <a>conforming verifier
+implementations</a> can run when verifying a <a>verifiable credential</a> or a
+<a>verifiable presentation</a>. This algorithm takes a byte sequence (bytes
+<var>inputBytes</var>) and a media type (string <var>inputMediaType</var>) as
+inputs, and produces an object (<var>verified</var>) that contains the
+following:
+        </p>
+
+        <ul>
+          <li>
+a verification status (boolean <var>status</var>)
+          </li>
+          <li>
+a verified document (object <var>document</var>)
```

Comment:
The key to this being a valuable normative section is to be assured that when verified is true, the document here is a conformant document. 

I think this is probably implied, but we need to be explicit here, because later this document might end up either containing or not containing `proof`.

And in the case of proof chains and proof sets used by data integrity, the details around this property matter.

I think its correct to say that `proof` MAY be present in all conforming documents at the time we are discussing this.